### PR TITLE
Customer side

### PR DESCRIPTION
**Pull request by Balsam-Faysal**
#30 create pages of customer side with their styles and backend routes

### CODE REVIEW

File: server/controllers/api/checkFlight.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import models from '../../database/models/index';
+
+export default(req, res) => {
+  const { textInputValueFlightNo } = req.body;
+  models.Flights.findOne({ where: { flight_no: textInputValueFlightNo } }).then((result) => {
+    if (result && result.dataValues) {
+      res.send({ isAvailable: true, flight_no: result.dataValues.flight_no });
+      console.log(result.dataValues.flight_no);
```
Comment (NoureldeanSaed) [inline server/controllers/api/checkFlight.js]: Remove unnecessary logging.

File: server/controllers/api/updateFlight.js [hunk 2]
```diff
@@ -1,11 +1,12 @@
 import models from '../../database/models/index';
 
 export default (req, res, next) => {
-  const flightNumber = req.params.flightId;
+  const flightNumber = req.params.flightId || req.body.flightNo;
 
   models.Flights.findOne({ where: { flight_no: flightNumber } }).then((result) => {
     let resultFlightInformation = {};
     resultFlightInformation = result.dataValues;
+    console.log(resultFlightInformation);
```
Comment (NoureldeanSaed) [inline server/controllers/api/updateFlight.js]: Remove logs.

File: server/controllers/api/updateFlight.js [hunk 3]
```diff
@@ -1,11 +1,12 @@
 import models from '../../database/models/index';
 
 export default (req, res, next) => {
-  const flightNumber = req.params.flightId;
+  const flightNumber = req.params.flightId || req.body.flightNo;
 
   models.Flights.findOne({ where: { flight_no: flightNumber } }).then((result) => {
     let resultFlightInformation = {};
```
Comment (NoureldeanSaed) [inline server/controllers/api/updateFlight.js]: Why not `const resultFlightInfo = result.dataValues;` ?

**Comment by Balsam-Faysal** [comment]
I closed this PR @NoureldeanSaed

**Pull request closed without merging**